stan/math/fwd/scal/fun/rising_factorial.hpp

```diff
@@ -11,35 +11,14 @@ namespace stan {
     template<typename T>
     inline
     fvar<T>
-    rising_factorial(const fvar<T>& x, const fvar<T>& n) {
-      T rising_fact(rising_factorial(x.val_, n.val_));
-      return fvar<T>(rising_fact,
-                     rising_fact * (digamma(x.val_ + n.val_)
-                                    * (x.d_ + n.d_) - digamma(x.val_) * x.d_));
-    }
-
-    template<typename T>
-    inline
-    fvar<T>
-    rising_factorial(const fvar<T>& x, double n) {
+    rising_factorial(const fvar<T>& x, int n) {
       using boost::math::digamma;
```

Comment:
same comment on `boost::math::digamma`---just include San's primitive. Then it's all in namespace anyway.